src/components/Card.js

```diff
@@ -12,34 +17,58 @@ export default class Card {
       .content
       .querySelector('.card')
       .cloneNode(true);
-
     return cardElement;
   }
 
+  _renderLikes() {
+    if (this._likeCountElement) {
+      this._likeCountElement.textContent = this._likes.length;
+    }
+  }
+
+  _setEventListeners() {
+    this._cardImage.addEventListener('click', () => {
+      this._handleCardClick(this._name, this._link);
+    });
+
+    this._likeButton.addEventListener('click', () => {
+      const isLiked = this._likeButton.classList.contains('card__like-button_active');
+      this._likeButton.classList.toggle('card__like-button_active'); 
+      localStorage.setItem(this._id, JSON.stringify(this._likeButton.classList.contains('card__like-button_active')));
+      this._handleLikeClick(this._id, !isLiked, this);
+    });
+    
+
+    this._deleteButton.addEventListener('click', () => {
+      this._handleDeleteClick(this._id, this._element);
+    });
+  }
+
+
+  updateLikes(newLikes) {
+    this._likes = newLikes;
+    this._renderLikes();
```

Comment:
Please, add `this._likeButton.classList.toggle('card__like-button_active'); ` here


```js
  updateLikes(newLikes) {
    this._likes = newLikes;
    this._renderLikes();
    this._likeButton.classList.toggle('card__like-button_active'); 
  }

```